src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -8,22 +8,10 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <!-- Default Version for dev -->
     <Version>99.99.99</Version>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
There's still some of the properties you added in the root (Company, Authors, etc.) in here. Make sure you go through all the ones you moved and remove them from here too.